### PR TITLE
Chore: Re-write imports to be ES6

### PR DESCRIPTION
**Pull request by Pezmc**
## Description

Re-writes all use of '@/' in the front-end codebase to use a relative import.

Please see reasoning in https://github.com/flowforge/flowforge/issues/1908

Was replaced using a [script](https://gist.github.com/Pezmc/9d78ce0a6fea39e80b501e8c48b24586).

## Related Issue(s)

Part of https://github.com/flowforge/flowforge/issues/1908

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [-]  Suitable unit/system level tests have been added and they pass
 - [-] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by Pezmc** [comment]
Rebased onto main to pull in https://github.com/flowforge/flowforge/pull/1846 so [356516c](https://github.com/flowforge/flowforge/pull/1909/commits/356516c66bb79aac779723ce75734ea7c8866f14) can enable a lint rule that requires all files to resolve.

File: .eslintrc [hunk 1]
```diff
@@ -15,12 +15,22 @@
...
+            }
+        },
+        "import/core-modules": [ "lottie-web-vue" ],
         "node": {
             "allowModules": [
                 "flowforge-test-utils", // exists at test/node_modules/flowforge-test-utils
                 "./config",
                 "promptly",
-                "@core"
+                "lottie-web-vue"
```
Comment (Steve-Mcl) [inline .eslintrc:33]: @Pezmc what is the purpose of the addition of `lottie-web-vue` in this PR?

Comment (Pezmc) [reply .eslintrc:33]: `lottie-web-vue` doesn't export it's module correctly, so the require check fails to find it and throws errors without marking it to ignore.

**Review by Steve-Mcl — APPROVED** [review]
@Pezmc I am personally very happy to see the improvements to development experience this PR brings. Review looks good. Cant spot any missing refs & tests pass - gets my approval.

**Comment by Steve-Mcl** [comment]
@Pezmc might be worth a mention in dev channel (do a pull on main) due to many small changes.

**Pull request merged**